content/guides/userguides/terra-export.md

```diff
@@ -0,0 +1,123 @@
+---
+path: "/guides/userguides/consumer-vignettes/export-to-terra"
+date: "2019-09-17"
+title: "Exporting HCA Data to Terra"
+subTitle: ""
+---
+
+# Exporting search results from the HCA Data Explorer to Terra
+
+In this tutorial, you'll learn how to send search results from the HCA Data
+Explorer to Terra and how to run a basic workflow with that data.
+
+We'll assume that you have some familiarity with the tools aforementioned.
+In particular, you'll want to be acquainted with the content in these
+tutorials:
+
+- [Accessing HCA Data and Metadata](quick-start-guide)
```

Comment:
This link is broken

Reply:
Looks like `quick-start-guide` is missing from the links list at the bottom of this document